src/NexusMods.App.UI/Pages/LibraryPage/TreeDataGridLibraryResources.axaml

```diff
@@ -136,21 +137,81 @@
                                                              Fill="StrongAlt"
                                                              LeftIcon="{x:Static icons:IconValues.Add}"
                                                              ShowIcon="Left"
-                                                             IsVisible="{CompiledBinding !IsInstalled.Value}"/>
+                                                             IsVisible="{CompiledBinding !IsInstalled.Value}" />
                                     <StackPanel x:Name="InstalledStack"
                                                 Orientation="Horizontal"
                                                 IsVisible="{CompiledBinding IsInstalled.Value}">
-                                        <icons:UnifiedIcon Value="{x:Static icons:IconValues.Check}"/>
-                                        <TextBlock Text="{CompiledBinding ButtonText.Value}"/>
+                                        <icons:UnifiedIcon Value="{x:Static icons:IconValues.Check}" />
+                                        <TextBlock Text="{CompiledBinding ButtonText.Value}" />
                                     </StackPanel>
                                 </Panel>
                             </DataTemplate>
                         </controls:ComponentTemplate.DataTemplate>
                     </controls:ComponentTemplate>
                 </controls:ComponentControl.ComponentTemplate>
             </controls:ComponentControl>
+           
+            <!-- more options -->
+            <controls:StandardButton
+                ToolTip.Tip="More options"
+                x:Name="HamburgerMenuButton"
+                LeftIcon="{x:Static icons:IconValues.MoreVertical}"
+                ShowIcon="Left"
+                ShowLabel="False"
+                Fill="None">
+                <controls:StandardButton.Flyout>
```

Comment:
This can be empty if dealing with a local file mod (not from nexus).
What is the expected behaviour of the button in that case?